ably/rest/auth.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, ably, options):
             raise ValueError("Can't authenticate via token, must provide "
                              "auth_callback, auth_url, key, token or a TokenDetail")
 
-    def authorize(self, token_params=None, auth_options=None, force=False):
+    def _authorize(self, token_params=None, auth_options=None, force=False):
```

Comment:
Not overly keen on the cryptic name for this method, especially as I believe this method is public.  Firstly can this method be private unless it's needed to be public.  Secondly, could the name be more descriptive of what it does such as https://github.com/ably/ably-ruby/blob/4008873524411d368321bd489fff90e9118c3132/lib/ably/auth.rb#L476?